CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Fixes
 
-* **Header:** fixed hover effects on header logo.
+* **Header:** fixed hover effects on header logo. ([#413](https://www.github.com/iTwin/iTwinUI-react/issues/413)) ([a360a8e](https://www.github.com/iTwin/iTwinUI-react/commit/a360a8e5db6c98017d151d7f7e6d4e3ba768eb04))
```

Comment:
Maybe write `Fixed` (uppercased) as in other entries.

Reply:
changed